src/bag/bag_loader.py

```diff
@@ -0,0 +1,116 @@
+# the tables we get from the BAG dump do not contain primary keys. These
+# are required if we want to reference them in our Django models.
```

Comment:
Deze comment staat hier nogal verloren. Slaat hij op deze hele file? Of op een bepaald deel? Of misschien is het een TODO (nu niet neem ik aan, maar dat is voor de toekomstige lezer niet duidelijk lijkt me).